perf/benchmark/README.md

```diff
@@ -224,15 +224,11 @@ Calls to Istio's Mixer component (policy and telemetry) adds latency to the side
     python ./update_mesh_config.py enable_mixer /tmp/meshconfig.yaml  | kubectl -n istio-system apply -f -
     ```
 
-## [Optional] Enable Telemetryv2
+## [Optional] Enable Mixerless Telemetry:
```

Comment:
Please change the name back to telemetry v2.
Otherwise it is ok.

Reply:
sure